src/lib.rs

```diff
@@ -1,7 +1,14 @@
-/**
+/*
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.*
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/.>*
```

Comment:
Rustfmt wanted to do that with the link for some reason.

I removed the asterisks because at some point this comment appeared in the doc. But this turned out to be caused by sth else